pages/linux/playerctl.md

```diff
@@ -0,0 +1,27 @@
+# playerctl
+
+> Utility to control different media players.
+
+- Toggle play:
+
+`playerctl play-pause`
+
+- Next media:
+
+`playerctl next`
+
+- Previous media:
+
+`playerctl previous`
+
+- List all players:
+
+`playerctl --list-all`
+
+- Send a command to a specific player:
```

Comment:
I guess by default it picks one at random then?

Reply:
By default it uses the first player it can find.

Reply:
Ah, ok. Looks ok to me then!